barretenberg/cpp/src/barretenberg/stdlib/primitives/group/cycle_group.test.cpp

```diff
@@ -1144,6 +1144,9 @@ TYPED_TEST(CycleGroupTest, TestBatchMulIsConsistent)
 TYPED_TEST(CycleGroupTest, MixedLengthScalarsIsNotSupported)
```

Comment:
This is a test added by me in a previous PR to ensure that attempting to use scalars of different sizes breaks gracefully. (This was previously claimed to be supported but was actually broken and not used anyway). Here I'm just updating it to reflect that we no longer allow anything other than 256 bits in method `from_256_bit_witness` (formerly `from_witness_bitstring()`)